Ix/Archon/TowerField.lean

```diff
@@ -30,4 +30,10 @@ instance : ToString TowerField where
 @[extern "c_rs_add_u128_in_binary_field"]
 opaque addUInt128InBinaryField : @& UInt128 → @& UInt128 → TowerField → UInt128
 
+@[extern "c_rs_mul_u128_in_binary_field"]
+opaque mulUInt128InBinaryField : @& UInt128 → @& UInt128 → UInt128
+
+@[extern "c_rs_inv_u128_in_binary_field"]
+opaque invUInt128InBinaryField : @& UInt128 → UInt128
```

Comment:
I see that you're not requiring a `TowerField` as a parameter. Is it necessary for addition, as I did above?

Reply:
I think it's necessary for all functions, including the ones you added 🤔

Reply:
Oh, nvm! Since these are fields, they're closed under addition and multiplication!